packages/flutter/pubspec.yaml

```diff
@@ -10,29 +10,29 @@ dependencies:
   flutter:
     sdk: flutter
 
-  parse_server_sdk: ^3.1.2
```

Comment:
There can be no changes in the flutter package in a dart package release.

Reply:
This change does not affect the Flutter codes, and after the release of the Dart version, the Flutter version will be released immediately

Reply:
Still this is a dart release, so there cannot be any changes in the flutter package.

Reply:
No, it is not necessary. It can be added in another PR related to Flutter. I will return this part

Reply:
I returned to the previous version